### PR TITLE
FIX: build using tf sysconfig

### PR DESCRIPTION
**Pull request by seanpmorgan**
This will make the lookup of shared library name OS agnostic.

### CODE REVIEW

File: tf_dependency/tf_configure.bzl [hunk 1]
```diff
@@ -183,19 +185,22 @@ def _tf_pip_impl(repository_ctx):
     )
 
     tf_shared_library_dir = repository_ctx.os.environ[_TF_SHARED_LIBRARY_DIR]
-    tf_shared_library_path = "%s/libtensorflow_framework.so.2" % tf_shared_library_dir
+    tf_shared_library_name = repository_ctx.os.environ[_TF_SHARED_LIBRARY]
```
Comment (facaiy) [inline tf_dependency/tf_configure.bzl]: Can we use `_TF_SHARED_LIBRARY_NAME` for consistency?

File: README.md [hunk 2]
```diff
@@ -62,8 +62,8 @@ To install the latest version, run the following:
 ```
 pip install tensorflow-addons
 ```
-
-**Note:** You will also need [TensorFlow 2.0 or higher](https://www.tensorflow.org/alpha). 
+ 
+**Note:** You will also need `tensorflow==2.0.0.a0` [installed](https://www.tensorflow.org/alpha)
```
Comment (facaiy) [inline README.md]:
```suggestion
**Note:** You will also need [`tensorflow==2.0.0.a0`](https://www.tensorflow.org/alpha) installed.
```

File: README.md [hunk 3]
```diff
@@ -76,6 +76,8 @@ import tensorflow_addons as tfa
 You can also install from source. This requires the [Bazel](
 https://bazel.build/) build system.
 
+**Note:** If building from master you must install `tf-nightly-2.0-preview` in the process
```
Comment (facaiy) [inline README.md]:
```suggestion
**Note:** If building from master you must install `tf-nightly-2.0-preview` in the process.
```

File: tf_dependency/tf_configure.bzl (hunk 1 above)
Comment (seanpmorgan) [reply tf_dependency/tf_configure.bzl]: Yeah makes sense. Thanks.

**Pull request merged**